roles/ldap_kerberos/templates/krb5.conf.j2

```diff
@@ -8,6 +8,7 @@ includedir /etc/krb5.conf.d/
 [libdefaults]
   dns_lookup_realm = false
   ticket_lifetime = 24h
+  renew_lifetime = 7d
```

Comment:
The effective ticket renew lifetime is the min of renew_lifetime and max_renewable_life. Should theses 2 variables be synchronized?

Reply:
We don't really care here, the most important is that the tickets are renewable. It's not a requirement for them to be synchronized.

Reply:
As max_renewable_life is variablized, ticket_lifetime and renew_lifetime should be settled in the same way . A comment for the guideline utilization of renew_lifetime and max_renewable_life will help for a  better control of ticket renewal lifetime.

Reply:
Ok, I added templatisation of the `renew_lifetime` parameter, added some information in readme about it.

Note that I did not variabilize `ticket_lifetime` as it is not in the scope of this PR.

Reply:
It's good for me.

Reply:
Thanks, I'm going to merge then